tv_apps/smart-system/js/app_usage_metrics.js

```diff
@@ -454,6 +458,17 @@
       this.currentApp = e.detail;
       this.currentAppStartTime = now;
       break;
+    case APPLOADED:
+      // For "send to TV" feature only.
+      // For now, notification-receiver launches every time it receive send to
+      // TV request. So invocation count of notification-receiver should be
+      // equal to the count of send to TV usage count.
```

Comment:
Please also mention that "notification-reveiver" will never become foreground so we can't track it by "appopened".